### PR TITLE
Chore/misc changes

### PR DESCRIPTION
**Pull request by PedroMiguelSS**
Changes:

- Update `TypeGroup`, `TypeOption` and `TextInput` components
- Update `TypeGroup` to be responsive
- Remove unnecessary `console.log`
- Update `AvatarPicker` `font-size` units
- Update `TypeOption` css properties:
  - `background` to `background-color`
  - `font-size` units
- Update `FlowModalContents` component
  - `ModalClose` is not visible anymore on full layouts
  - Current step on full layout occupies 100% of `height`
- Update ModalClose font-size units

### CODE REVIEW

File: src/components/modal-base/Modal.css [hunk 1]
```diff
@@ -92,7 +92,7 @@
 .modalBodyOpen {
     overflow: hidden;
 
-    & .modalPortal {
+    & .modalPortal:not(:empty) {
```
Comment (dominguesgm) [inline src/components/modal-base/Modal.css:95]: Thanks for including this one 😄

**Review by dominguesgm — APPROVED** [review]
👍

File: src/components/type-group/TypeGroup.js [hunk 2]
```diff
@@ -20,10 +20,10 @@ const TypeGroup = ({ children, className, name, onSelect, selectedId }) => {
 
 TypeGroup.propTypes = {
     children: PropTypes.node.isRequired,
-    onSelect: PropTypes.func.isRequired,
     name: PropTypes.string.isRequired,
     selectedId: PropTypes.string,
     className: PropTypes.string,
+    onSelect: PropTypes.func,
```
Comment (satazor) [inline src/components/type-group/TypeGroup.js]: Shouldn't this be required?

File: src/components/type-group/type-option/TypeOption.css [hunk 3]
```diff
@@ -1,17 +1,19 @@
 @import "../../../styles/colors";
+@import "../../../styles/mixins/shared";
```
Comment (satazor) [inline src/components/type-group/type-option/TypeOption.css]: Can you rename `shared` to shapes? All mixins are shared.

File: src/components/type-group/type-option/TypeOption.js [hunk 4]
```diff
@@ -18,7 +21,8 @@ class TypeOption extends Component {
                         checked={ selected }
                         disabled={ !selectable }
                         onChange={ this.handleInputChange }
-                        onClick={ this.handleClick } />
+                        onClick={ this.handleClick }
```
Comment (satazor) [inline src/components/type-group/type-option/TypeOption.js]: I think we can remove the `onClick`.

File: src/components/type-group/type-option/TypeOption.js [hunk 5]
```diff
@@ -33,7 +37,7 @@ class TypeOption extends Component {
     }
 
     handleInputChange = () => {
-        this.props.onSelect(this.props.id);
+        this.props.onSelect && this.props.onSelect(this.props.id);
```
Comment (satazor) [inline src/components/type-group/type-option/TypeOption.js:39]: I would mark `onSelect` as mandatory in propTypes and remove the check here.

File: src/components/type-group/TypeGroup.js (hunk 2 above)
Comment (PedroMiguelSS) [reply src/components/type-group/TypeGroup.js]: Yes, it should be. Done!

File: src/components/type-group/type-option/TypeOption.js (hunk 4 above)
Comment (PedroMiguelSS) [reply src/components/type-group/type-option/TypeOption.js]: You're right. I'm already spreading the input props.

**Pull request merged**